stable/nginx/nginx/templates/NetworkPolicy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: policy-demo 
+  namespace: default
```

Comment:
```suggestion
```

I think we don't want this the policy needs to exist in the correct SLATE group namespace.

SLATE API should just handle this during the application install process? Perhaps, @cnweaver you can confirm?